app/bundles/LeadBundle/Tests/EventListener/CampaignSubscriberFunctionalTest.php

```diff
@@ -464,47 +464,6 @@ private function createCampaign(array $contactIds): Campaign
         $this->em->persist($event);
         $this->em->flush();
 
-        $event2 = new Event();
```

Comment:
How is removal of this event from the test related to the PR?

Reply:
This was happened maybe at time of conflict resolution. Fixed.